components/GoToTopButton/GoToTopButton.styled.js

```diff
@@ -0,0 +1,13 @@
+import styled from "styled-components";
+
+export const IconButton = styled.button`
+  position: fixed;
+  font-size: 30px;
+  bottom: 70px;
+  right: 10px;
+  background-color: transparent;
+  border: none;
+  cursor: pointer;
+  z-index: 1;
+  display: ${(props) => (props.showButton ? "" : "none")};
```

Comment:
instead of ternary operation display: ${(props) => (props.showButton ? "" : "none")};
you can use display: ${(props) => (!props.showButton && "none")};

Reply:
Thanks for the hint. I'll make the switch.